h/templates/activity/search.html.jinja2

```diff
@@ -33,26 +33,20 @@
 {# Card displaying statistics about a bucket of annotations. #}
 {% macro search_bucket_stats(bucket) %}
 <div class="search-bucket-stats">
-  {% if bucket.uri %}
+  {% if bucket.uri and bucket.incontext_link(request) %}
     <div class="search-bucket-stats__key">
-        URL
-    </div>
-    <div class="search-bucket-stats__val search-bucket-stats__url">
-        <div><a class="search-bucket-stats__link"
-                href="{{ bucket.uri }}">{{ pretty_link(bucket.uri) }}</a></div>
-        {% if bucket.incontext_link(request) %}
-        <div>
-          <a class="search-bucket-stats__incontext-link"
-             href="{{ bucket.incontext_link(request) }}"
-             target="_blank">
-             Visit annotations in context {{ svg_icon('up-right-arrow', css_class='search-bucket-stats__incontext-icon') }}
-           </a>
-        </div>
-        {% endif %}
+      {{ svg_icon('up-right-arrow', css_class='search-bucket-stats__icon') }}
+      <a class="search-bucket-stats__incontext-link"
+         href="{{ bucket.incontext_link(request) }}"
+         target="_blank">
+         Visit annotations in context
```

Comment:
Should this be inside `{% trans %} ... {% endtrans %}` tags? I don't think we have any agreed on i18n standards, and of course we don't localise h into any other languages, but I've been trying to remember to use `_()` in Python and `{% trans %}` in Jinja2 for user-visible strings. There's some docs on `{% trans %}` etc here: http://docs.ckan.org/en/latest/contributing/string-i18n.html#internationalizating-strings-in-jinja2-templates